Sources/TensorFlow/Initializers.swift

```diff
@@ -552,6 +552,94 @@ public extension Tensor where Scalar: TensorFlowFloatingPoint {
     }
 }
 
+public extension Tensor where Scalar: TensorFlowFloatingPoint {
+    /// Creates a tensor with the specified shape by performing He (Kaiming) uniform initialization.
+    ///
+    /// It draws random samples from a uniform distribution between `-limit` and `limit`
+    /// generated by the default random number generator, where `limit` is
+    /// `sqrt(6 / fanIn)` and `fanIn` represents the number of input features multiplied by the 
+    /// receptive field size.
+    ///
+    /// Reference: ["Delving Deep into Rectifiers: Surpassing Human-Level Performance on ImageNet 
+    /// Classification"](https://www.cv-foundation.org/openaccess/content_iccv_2015/papers/He_Delving_Deep_into_ICCV_2015_paper.pdf)
+    ///
+    /// - Parameters:
+    ///   - shape: The dimensions of the tensor.
```

Comment:
Could you please add a parameter doc comment for `seed`? Some other initializers have it.
```swift
    ///   - seed: The seed value.
```
Same elsewhere please.

Reply:
This was also missing from the Glorot uniform initializers, so I've added it into the new ones as well as the old.